packages/boot/src/booters/booter-utils.ts

```diff
@@ -39,22 +40,25 @@ export function isClass(target: any): target is Constructor<any> {
 }
 
 /**
- * Returns an Array of Classes from given files. Works by requiring the file,
- * identifying the exports from the file by getting the keys of the file
- * and then testing each exported member to see if it's a class or not.
+ * Returns a promise of an Array of Classes from given files. Works by requiring
+ * the file, identifying the exports from the file by getting the keys of the
+ * file and then testing each exported member to see if it's a class or not.
  *
  * @param files - An array of string of absolute file paths
  * @param projectRootDir - The project root directory
- * @returns An array of Class constructors from a file
+ * @returns A promise of an array of Class constructors from a file
  */
-export function loadClassesFromFiles(
+export async function loadClassesFromFiles(
   files: string[],
   projectRootDir: string,
-): Constructor<{}>[] {
+): Promise<Constructor<{}>[]> {
   const classes: Constructor<{}>[] = [];
   for (const file of files) {
     debug('Loading artifact file %j', path.relative(projectRootDir, file));
-    const moduleObj = require(file);
+    // This is to avoid that compilation transform the import into a require...
+    const moduleObj = await Function(
+      `return import("${pathToFileURL(file).toString()}");`,
```

Comment:
Let’s use import() directly.

Reply:
if you use directly import it will be transpilled as a riquire by lb-tsc